server/images.go

```diff
@@ -1162,3 +1164,24 @@ func verifyBlob(digest string) error {
 
 	return nil
 }
+
+var isContainer = ""
+
+func init() {
```

Comment:
`init()` is hard to test. Consider using something like `sync.OnceValue` instead

```go
var isContainer = sync.OnceValue(func() T {
  return ...
})
```